cluster-install/install.sh

```diff
@@ -583,7 +583,7 @@ prompt_user() {
     if [ "$DISABLE_PROMPTS" == "true" ]; then return 0; fi
 
     while true; do
-        read -p "$prompt_msg" -r "${var_name?}"
+        read -p "${prompt_msg% } " -r "${var_name?}"
```

Comment:
Minor fix to how prompts are displayed; this ensures there will always be a space at the end of the prompt, whether it was manually added by the consumer of the function or not. For example, here's how it would be if the consumer forgot to add the space:
`Please enter the namespace:astra`
With the fix:
`Please enter the namespace: astra`
And if they did include the space, it's a no-op.